### ISSUE TITLE
LV2 crashes

### ISSUE THREAD

**Issue by Bleuzen** [flathub/org.freedesktop.LinuxAudio.Plugins.ChowTapeModel#8]
When loading the LV2 plugin in Ardour, it crashes and prints the following:

`/usr/include/c++/12.1.0/bits/stl_vector.h:1123: std::vector<_Tp, _Alloc>::reference std::vector<_Tp, _Alloc>::operator[](size_type) [with _Tp = juce::SmoothedValue<float, juce::ValueSmoothingTypes::Multiplicative>; _Alloc = std::allocator<juce::SmoothedValue<float, juce::ValueSmoothingTypes::Multiplicative> >; reference = juce::SmoothedValue<float, juce::ValueSmoothingTypes::Multiplicative>&; size_type = long unsigned int]: Assertion '__n < this->size()' failed.`

**Comment by hfiguiere** [comment]
oh that. I can look at a patch. tl;dr BAD C++ (it's the language that suck)

**Comment by hfiguiere** [comment]
the original developer probably didn't catch it because by default assertions aren't enabled, and this UB coincidentally work. On flatpak they are.

**Comment by hfiguiere** [comment]
It's not a random "access the array of zero size" I have seen elsewhere.

I commented on the upstream bug:
https://github.com/jatinchowdhury18/AnalogTapeModel/issues/283#issuecomment-1382912328

**Comment by hfiguiere** [comment]
tl;dr the LV2 is broken upstream at the moment.
